lib/reporting/reports/enterprise_fee_summary/enterprise_fees_with_tax_report_by_producer.rb

```diff
@@ -1,17 +1,15 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ClassLength
 module Reporting
   module Reports
     module EnterpriseFeeSummary
       class EnterpriseFeesWithTaxReportByProducer < ReportTemplate
-        attr_accessor :permissions, :parameters
+        attr_accessor :permissions
 
         def initialize(user, params = {}, render: false)
           super(user, params, render: render)
-          @parameters = Parameters.new(p || {})
-          @parameters.validate!
           @permissions = Permissions.new(user)
-          @parameters.authorize!(@permissions)
```

Comment:
When I used `git rebase -i HEAD~X`, I was trying to keep the history of the PR clean.
This also makes it difficult for me to keep track of some changes.
Sorry about that. From now, I'll avoid `rebase` after getting a review (especially on big PRs)

Reply:
I'm not saying to never do a `rebase`, in fact my point is to keep the git history clean before requesting a review. (but carefully, because you can fall into a hole, which has happened to me a lot).

As you say, I think it's usually good to avoid the rebase after a review, so that we can discuss and review the iterations.